config/clientparameters.go

```diff
@@ -22,6 +22,7 @@ type ClientParameters struct {
   JobParams           HuntingParameters     `json:"job"`
   DocsUrl             string                `json:"docsUrl"`
   CheatsheetUrl       string                `json:"cheatsheetUrl"`
+  ReleasenotesUrl     string                `json:"releaseNotesUrl"`
```

Comment:
The N on the `ReleasenotesUrl` should be capitalized since the json alias uses that Proper casing format.

Reply:
Good catch, fixed in latest push!